package.json

```diff
@@ -4,6 +4,9 @@
   "description": "Data Federation console",
   "author": "SanjalKatiyar <skatiyar@redhat.com>",
   "private": true,
+  "workspaces": [
+    "packages/*"
+  ],
```

Comment:
no need to set workspaces... we are not maintaining sub-repos under dfr-console...